server/app/boards.go

```diff
@@ -494,11 +494,48 @@ func (a *App) DeleteBoard(boardID, userID string) error {
 }
 
 func (a *App) GetMembersForBoard(boardID string) ([]*model.BoardMember, error) {
-	return a.store.GetMembersForBoard(boardID)
+	members, err := a.store.GetMembersForBoard(boardID)
+	if err != nil {
+		return nil, err
+	}
+
+	for i, m := range members {
+		if !m.SchemeAdmin {
+			board, err := a.store.GetBoard(m.BoardID)
```

Comment:
Do we need to fetch the board for each loop here?  I think fetching it once outside the loop would suffice.

Reply:
Thanks...